src/server/game/DataStores/DB2Stores.cpp

```diff
@@ -516,6 +520,7 @@ namespace
     std::unordered_map<uint32, uint8> _pvpItemBonus;
     PvpTalentSlotUnlockEntry const* _pvpTalentSlotUnlock[MAX_PVP_TALENT_SLOTS];
     std::unordered_map<uint32, std::vector<QuestLineXQuestEntry const*>> _questsByQuestLine;
+    std::unordered_map<uint32, std::vector<QuestLineXQuestEntry const*>> _questsByQuestXLine;
```

Comment:
I think _questXLinesByQuest is a better name (you store all QuestLineXQuestEntry for each quest)